### ISSUE TITLE
Android Mobile App - Timeline scrolling not as smooth on 1.99.0.

### ISSUE THREAD

**Issue by m-paulus** [immich-app/immich#8197]
### The bug

Scrolling through any timeline is noticeably more sluggish and jumpy since the latest update to 1.99.0. 
I frequently feel haptic feedback while swiping to scroll, almost every time I touch the screen. The scroll start is delayed and then quickly jumps to catch up. I suspect that it is because the app is trying to detect whether I'm holding down for the new drag-to-select feature. 

### The OS that Immich Server is running on

Debian

### Version of Immich Server

1.99.0

### Version of Immich Mobile App

1.99.0

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
n/a
```

### Your .env content

```Shell
n/a
```

### Reproduction steps

```bash
1. Open the mobile app on Android
2. Start scrolling through the gallery
```

### Additional information

my phone model is Google Pixel 6a

**Comment by alextran1502** [comment]
Fixed in #8166

**Comment by m-paulus** [comment]
you guys are amazing
